doc/api/api_changes/2017-09-22-AL-removal-of-deprecated-features.rst

```diff
@@ -0,0 +1,17 @@
+Removal of deprecated features
+``````````````````````````````
+
+The ``matplotlib.finance``, ``mpl_toolkits.exceltools`` and
+``mpl_toolkits.gtktools`` modules have been removed.
+
+The ``mpl_toolkits.mplot3d.art3d.iscolor`` function has been removed.
+
+The ``Axes.get_axis_bgcolor``, ``Axes.set_axis_bgcolor``,
+``Bbox.update_from_data``, ``Bbox.update_datalim_numerix``,
+``MaxNLocator.bin_boundaries`` methods have been removed.
+
+The ``bgcolor`` keyword argument to ``Axes`` has been removed.
+
+The ``spectral`` and ``Vega*`` colormaps have been removed.
```

Comment:
`Vega` got renamed right? Might be worth mentioning that here.

Reply:
done